### DISCUSSION TITLE
When are issues tagged as "bug"? What does an issue need to provide to prove it is a bug?

### DISCUSSION THREAD

**Discussion by Fabrizio-Caruso** [cc65/cc65#1635] [General]
Hi everyone,
I would like to understand what is needed to have issues marked as "bug". 
For example I have opened https://github.com/cc65/cc65/issues/1584 where I reproduce a misbehavior in the TGI library for the Atari Lynx. Unless I have overlooked something, it looks like I may have found a real bug. I may try myself to look into this but I may lack the skills.
I am not asking for a quick fix from anyone but maybe having a tag could at least clarify whether it is a bug for other people or not.
Is an example that reproduce a wrong behavior, not enough to get a "bug" tag?
Or is the tag "bug" used to mean something else? 
Personally I think a bug should be tagged as such even though there is no quick solution or it has low priority because it is a way to document an issue (even a minor one).

**Comment by polluks** [comment]
Well, it's more a question of expertise. You find much knowledge for src/ but less for libsrc/. If you have an uncommon platform you have to wait for the tag...

**Comment by Fabrizio-Caruso** [comment]
@polluks I am not talking about fixing it. I am talking about visible behavior. I am not asking anyone to jump and fix it asap.
Whatever behavior, TGI apis are supposed to behave the same across all targets.
I have reported and reproduced a TGI misbehavior on a target X. If you do the same on a target Y, you will get a different behavior. 
No need to know anything about target X and Y to agree on the bug. Fixing it is a different story.
Moreover the behavior is usually considered "wrong" if it is not "consistent" (one color behaving differently than others). In this case it is worse, we have one color behaving differently than other colors only in one target.
I have already provided a minimal example. What else should I provide?
It is enough to load the binary into an emulator, and anyone can see what is wrong.

**Comment by mrdudz** [comment]
pardon the question, but can you not change the tag yourself? if not, i'll have to look into the setup and make it so

**Comment by Fabrizio-Caruso** [comment]
@mrdudz as far as I can see, I don't think I have the rights. I would like someone else confirm it. I mean confirm the observed behavior is wrong and not necessarily the technical cause. I am not asking for a quick fix by someone even though I would like to see this fixed one day in the future.

**Comment by mrdudz** [comment]
I'll have a look and change the permissions accordingly (and i agree it is a bug for that matter)

**Reply by Fabrizio-Caruso** [reply]
Thanks.

**Comment by mrdudz** [comment]
looks like its not possible to give that permission to non team members... that sucks :(

**Reply by Fabrizio-Caruso** [reply]
Thanks!

It is not so bad as this allows team members to decide whether an issue is a real bug or not.
I was confident about this very specific bug but in general people (including myself) open bug issues that may not be bugs. Someone with more expertise should (from time to time) look at the issues and tag the good ones and/or discard the bad ones.

By the way I have contributed (very minimally) to the project in the past with some little code from time to time to fix some bugs but I would not want to be an official team member because my contribution is minimal and I do not have enough Assembly expertise.
